examples/reverse-proxy/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     environment:
       # This tells Jaeger that it should run the UI with a given prefix.
       QUERY_BASE_PATH: /jaeger/prefix
+      QUERY_UI_BASE_PATH: /hub/jaeger-query/v1/jaeger/viewer
```

Comment:
how was this tested?

Reply:
@yurishkuro Testing the changes. Till now I had an issue with docker and I resolved it and testing it. I will update the  progress of this testing of my code-changes

Reply:
@yurishkuro When I tried to access this with the new updated UIBASEPATH : 

![image](https://github.com/jaegertracing/jaeger/assets/117595548/70c74a5c-69a6-4383-927c-e752ed85cd01)


IMO I have made related changes to access it through uibasepath and load files as intended. Am I missing anything related?

Can please review my code changes?